### DISCUSSION TITLE
Failure to use adaptive Isotropic Remeshing

### DISCUSSION THREAD

**Discussion by math-araujo** [cnr-isti-vclab/PyMeshLab#77] [Q&A] [answered]
Hey,

I'm trying to apply the [isotropic explicit remeshing](https://pymeshlab.readthedocs.io/en/latest/filter_list.html#remeshing_isotropic_explicit_remeshing) filter on a given mesh, using the `adaptive` parameter. The following code throws an PyMeshLabException on PyMeshLab, but on MeshLab (GUI) I'm able to apply the filter with adaptive remeshing flag set to true without any kind of preprocessing of the file. Am I doing something wrong? 
Thanks in advance! Also, thanks for the work on MeshLab and PyMeshLab, both are extremely helpful!

Python script:
```
from pathlib import Path
import pymeshlab

def minimal_example():
    mesh_set = pymeshlab.MeshSet()
    mesh_file = Path("subject_01_frontal1_depth10.obj")
    mesh_set.load_new_mesh(str(mesh_file))
    print(f"Starting remeshing...")
    #mesh_set.apply_filter("remeshing_isotropic_explicit_remeshing") # This line works
    mesh_set.apply_filter("remeshing_isotropic_explicit_remeshing", adaptive=True) # This line doesn't work...
    print(f"Done")
    mesh_set.save_current_mesh(mesh_file.stem + "_remesh.obj")
    
if __name__ == '__main__':
    minimal_example()
```

Output:
```
PyMeshLab 0.1.9 based on MeshLab 2020.12d
Starting remeshing...
Traceback (most recent call last):
File "remeshing.py", line 56, in <module>
mesh_set.apply_filter("remeshing_isotropic_explicit_remeshing", adaptive=True)
pymeshlab.pmeshlab.PyMeshLabException: Failed to apply filter: remeshing_isotropic_explicit_remeshing
Details: Missing Component
```

**Comment by alemuntoni** [comment, marked as answer]
I missed this thread because it should actually be an issue...
See #85

**Reply by math-araujo** [reply]
Thanks for the help! Sorry for reporting on the wrong place, I thought that I was misunderstanding the documentation.
